### PR TITLE
UI Tweaks

### PR DESCRIPTION
**Pull request by RDIL**
**!!!** Get ready for a long PR description - please read all of it for some things you will want to know

## Dependencies

* Latest react scripts
* Latest react helmet
* Latest prettier

### Finally fixed the inconsistencies with link elements

Before:

Screenshot 2020-04-24 at 9 18 07 PM

After:

Screenshot 2020-04-24 at 9 19 59 PM

* Fixed some devices doing really strange things because of broken shadows

Before (on less broken browsers):

Screenshot 2020-04-24 at 9 21 39 PM

(Notice how it is very dark on the first paper element but not nearly as dark on the second one)

After:

Screenshot 2020-04-24 at 9 22 59 PM

This shadow style is now reflected across the entire site.

(Yes, I did also put some margins on the table)

* Changed "profile" to "settings" for clarity (I read a book about designing user interfaces that basically says make everything as clear as possible - people associate settings with configuration)

Before:

Screenshot 2020-04-24 at 9 26 04 PM

After:
Screenshot 2020-04-24 at 9 24 12 PM

* Fixed whatever flexbox was doing when the sidebar is open

Before:

Screenshot 2020-04-24 at 9 29 49 PM

After:

Screenshot 2020-04-24 at 9 30 35 PM

## Codebase Enhancements

* More classes are now hooks because classes are out and hooks are in
* Removed lots of unused JSS classes from `withStyles` calls - its just dead code
* Changed a lot of the `<Icon>` usages to just importing the icon from `@material-ui/icons` which is better in cases such as times when the icons font fails to load
* I just purchased JetBrains WebStorm and so I gave it a spin with this and wow it found a lot of ways to improve existing code

-----

Signed-Off-By: Reece Dunham <me@rdil.rocks>

### CODE REVIEW

**Comment by RDIL** [comment]
Ping @fkorotkov for review when you have time 😄

File: src/components/account/UserProfile.tsx [hunk 1]
```diff
@@ -113,74 +110,66 @@ class UserProfile extends React.Component<Props> {
...
+                    style={{ cursor: 'pointer' }}
+                  >
+                    <TableCell>
+                      <Typography variant="h6">{organization.name}</Typography>
+                    </TableCell>
+                    <TableCell>
+                      <Tooltip title="Organization settings">
+                        <IconButton
+                          onClick={e => navigate(this.context.router, e, '/settings/github/' + organization.name)}
+                          className="pull-right"
```
[the remark this answers is not in the document]
Comment (RDIL) [reply src/components/account/UserProfile.tsx:132]: Yeah, I also noticed that.  I will take a look.

[the remark this answers is not in the document]
Comment (RDIL) [reply src/components/account/UserProfile.tsx]: Fixed!

**Comment by fkorotkov** [comment]
BTW I didn't notice 

> More classes are now hooks because classes are out and hooks are in

But overall it's looking good to me. Running it with the latest changes to verify the fix and will merge shortly.

**Comment by RDIL** [comment]
Yeah, I might have undone it at some point.  

Also, bundle size change overview:
```js
204.53 KB (+895 B)    build/static/js/8.ac4b5b8a.chunk.js
204.49 KB (+896 B)    build/static/js/9.05c05813.chunk.js
194.41 KB (+3.04 KB)  build/static/js/10.be7b6b7c.chunk.js
142.05 KB (-1.67 KB)  build/static/js/11.f0d77daf.chunk.js
65.51 KB (-1 B)       build/static/js/13.d576ae6f.chunk.js
33.6 KB (+570 B)      build/static/js/12.bf6dc654.chunk.js
21.62 KB (-6 B)       build/static/js/1.75b46439.chunk.js
15.78 KB (-17 B)      build/static/js/3.d8682f65.chunk.js
14.88 KB (+55 B)      build/static/js/14.17fe8f7e.chunk.js
11.43 KB (+2.25 KB)   build/static/js/16.04092de9.chunk.js
10.21 KB (-814 B)     build/static/js/15.2be98800.chunk.js
10.16 KB (+110 B)     build/static/js/main.d1305adc.chunk.js
9.37 KB (+1.12 KB)    build/static/js/17.d9056f28.chunk.js
8.68 KB (-8 B)        build/static/js/0.ddf8bbf7.chunk.js
6.55 KB (+419 B)      build/static/js/18.d71cf6e4.chunk.js
6.4 KB (-79 B)        build/static/css/10.9b621c35.chunk.css
5.66 KB (+469 B)      build/static/js/5.367de439.chunk.js
3.44 KB (+142 B)      build/static/js/22.8a02d725.chunk.js
3.36 KB (+37 B)       build/static/js/2.c91de07f.chunk.js
3.09 KB (-879 B)      build/static/js/19.f764d0ed.chunk.js
2.92 KB (+264 B)      build/static/js/20.cee2f9b8.chunk.js
2.72 KB (+9 B)        build/static/js/4.f3366fdb.chunk.js
1.74 KB (-8 B)        build/static/js/runtime-main.f8124d5f.js
1.29 KB               build/static/js/24.fbda5b1f.chunk.js
1.21 KB (+1 B)        build/static/js/23.f87adf6b.chunk.js
1.06 KB               build/static/css/13.5a2f7233.chunk.css
900 B (-3 B)          build/static/js/21.b826f5b7.chunk.js
473 B (+35 B)         build/static/css/main.583a84df.chunk.css
```

**Comment by RDIL** [comment]
Cheers

**Pull request merged**